src/app/courses-page/course-search/course-search.component.spec.ts

```diff
@@ -21,4 +23,21 @@ describe('CourseSearchComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('onClick method should be called', () => {
+    spyOn(component, 'onClick');
+    buttonEl = fixture.debugElement.query(By.css('.search-btn'));
+    fixture.detectChanges();
+    buttonEl.triggerEventHandler('click', null);
+    expect(component.onClick).toHaveBeenCalled();
+  });
+
+  // TODO: Should be replaced when logic to the method will be added
+  it('onClick method should show correct result', () => {
```

Comment:
you can skip this test if it's in todo)
use `xit` instead of `it`

Reply:
@SteveRusin xit is used.
Thanks)